<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="runTab">
     <attribute name="title">
      <string>실행</string>
     </attribute>
     <widget class="QPushButton" name="runAllBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>실행</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>정지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leftBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>좌측이동</string>
      </property>
     </widget>
     <widget class="QPushButton" name="firstPos">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>초기위치 이동</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rightBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>우측이동</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>실행 예약</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="viewTab">
     <attribute name="title">
      <string>화면보기</string>
     </attribute>
     <widget class="QLabel" name="firstLbl">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>tbl1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="firstLbl_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>tbl2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="firstLbl_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>tbl3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="firstLbl_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>tbl4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="infoTab">
     <attribute name="title">
      <string>QR 변경</string>
     </attribute>
     <widget class="QPushButton" name="runAllBtn_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>등록</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runAllBtn_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>변경/제거</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="statustable_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>291</width>
        <height>291</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>20</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Item no.</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>물품 정보</string>
     </attribute>
     <widget class="QTableWidget" name="statustable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>20</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Item no.</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
